packages/common/src/projects/HubProjects.ts

```diff
@@ -0,0 +1,248 @@
+import { IUserRequestOptions } from "@esri/arcgis-rest-auth";
```

Comment:
Note: I'm experimenting changing this to a class w static methods, to see if we can use that to enforce function contracts